st2common/st2common/services/action.py

```diff
@@ -315,6 +315,28 @@ def request_resume(liveaction, requester):
     return (liveaction, execution)
 
 
+def get_root_liveaction(liveaction_db):
```

Comment:
Unit test for this method?

Reply:
Ack that, and as mentioned in a [previous comment](https://github.com/StackStorm/st2/pull/3653#discussion_r141243809) I will test for a variety of parent depths to ensure this works as desired.

Reply:
Added in 5c9be2438